config/locales/de.yml

```
@@ -90,12 +90,21 @@ de:
     onboarding_telegram_form:
       heading: Mit Telegram teilnehmen
       text: Geben Sie bitte unten Ihren Namen ein, um die Anmeldung abzuschließen.
-      submit: Anmeldung abschließen
-    telegram_explained:
-      link: Mit Telegram teilnehmen
-      header: Mit Telegram teilnehmen
-      subheader: Melden Sie sich über den Button unten bei Telegram an, damit wir Ihnen Nachrichten senden können.
-      important_info: Kommen Sie zurück zu dieser Seite, nachdem Sie sich bei Telegram angemeldet haben. Nur dann können wir Ihre Registrierung für %{project_name} abschließen.
+      submit: Anmeldung fortsetzen
+    onboarding_telegram_link:
+      heading: Fast geschafft!
+      text: Verbinden Sie Ihr Telegram-Konto in der Telegram-App, um die Anmeldung abzuschließen.
+      link: Zur Telegram-App
+      copy: Link kopieren
+      help: Über den Button gelangen Sie direkt zum Telegram-Profil von „%{project_name}”. Alternativ können Sie einen Link kopieren und auf Ihrem Smartphone öffnen.
+      didnt_work: Hat das nicht funktioniert? Oder werden Sie nach einem Code gefragt?
+    onboarding_telegram_fallback:
+      heading: Fast geschafft!
+      text: Verbinden Sie Ihr Telegram-Konto in der Telegram-App, um die Anmeldung abzuschließen.
+      steps:
+        - "Finden Sie uns auf Telegram. Suchen Sie dazu nach <strong>%{telegram_handle}</strong>."
+        - "Wenn Sie unser Telegram-Konto gefunden haben, klicken Sie auf <strong>Start</strong>."
+        - "Geben Sie diesen Code ein, sobald sie danach gefragt werden: <strong>%{token}</strong>."
```

Let’s remove the period at the end so people can’t get confused over wether the period is part of the code or not.

```suggestion
        - "Geben Sie diesen Code ein, sobald sie danach gefragt werden: <strong>%{token}</strong>"
```